PrimeNumbers/PrimeNumber_join.cpp

```diff
@@ -402,22 +412,42 @@ class PrimeNumbers
             JOIN_TYPE = 1;
         }
 
-        if (mwaitx_cycle_count_used)
+        if (wait_count_used)
         {
-            if ((join_type < 3) || (join_type > 6))
+            if ((join_type < 3) || (join_type > 10))
             {
                 printf("Warning: '--mwaitx_cycle_count' is specified, but value will not be used for 'pause' wait type.\n");
```

Comment:
but now we don't have -mwaitx_cycle_count anymore, right?

Reply:
fixed. Replaced with `-wait_count`.